### ISSUE TITLE
Missing API-KEY notification showing constantly for websites that does not use Adyen

### ISSUE THREAD

**Issue by Seppo-Konttinen-Solteq** [Adyen/adyen-magento2#1999]
**Describe the bug**
We have multistore setup in which some of the websites uses Adyen while other has different payment methods. Now with new version 8.16.0 (was 8.10.0) in admin I constantly get notification for missing API-KEY for certain website(s) which is bit annoying and will raise questions from merchant. API-KEY is not set by purpose as that website is not using Adyen payment method.

**To Reproduce**
1. Upgrade version from 8.10.0 to 8.16.0 (while this was already introduced in 8.11.1)
2. Login to admin
3. Double check that you have all needed credentials set to websites that you use Adyen with
4. See error for missing API-KEY as admin system wide notification

**Expected behavior**
There should be enable/disable toggle per website so that Adyen can be disabled for specific websites thus no error message shown

**Magento version**
2.4.4.-p3

**Plugin version**
8.16.0

**Screenshots**
When logged in (this comes constantly
![image](https://user-images.githubusercontent.com/45000414/226538538-b669f641-d1f0-40c1-8641-9cd228963707.png)

This is also always shown
![image](https://user-images.githubusercontent.com/45000414/226538410-aa2d4764-a613-4b5b-a11a-36246866926e.png)

**Desktop (please complete the following information):**
 - iOS
 - Browser MS Edge
 - Version 111.0.1661.44 (Official build) (x86_64)

**Comment by maaarghk** [comment]
I think this is a dupe of #1947

**Comment by Seppo-Konttinen-Solteq** [comment]
@maaarghk yes sorry I was supposed to link change from release notes where I think it has happened:

https://github.com/Adyen/adyen-magento2/releases/tag/8.11.1

[PW-7508] - Fix system messages for missing API key and new version notifier by @candemiralp in https://github.com/Adyen/adyen-magento2/pull/1868

and yes this is duplicate for https://github.com/Adyen/adyen-magento2/issues/1947

**Comment by Morerice** [comment]
Hi @Seppo-Konttinen-Solteq,

As @maaarghk mentioned, this is a duplicate of #1947. I will close this issue and we will provide any updates on #1947 

Regards,
Jean
Adyen
